content-src/asrouter/templates/Trailhead/Trailhead.jsx

```diff
@@ -131,9 +133,39 @@ export class _Trailhead extends React.PureComponent {
     return str.value;
   }
 
+  addUtmParams(url, isCard = false) {
```

Comment:
Mmm.. A quick comment/jsdoc about how this modifies a URL passed in or returns a new URL would help.